src/App.tsx

```diff
@@ -130,6 +149,53 @@ function App() {
             <Link to="/settings">Settings</Link>
           </div>
           <div className="hidden mr-5 sm:block">{data.user.name}</div>
+          <div className="flex flex-column mr-3">
+            <button className="font-thin" onClick={() => setExcludeProfileSelectorOpen(!excludeProfileSelectorOpen)}>
+              {currentExcludeProfile.displayName === "default" ? "(default)" : currentExcludeProfile.displayName}{" "}
+              {excludeProfileSelectorOpen ? "▴" : "▾"}
+            </button>
+            {excludeProfileSelectorOpen && (
+              <div
+                style={{
+                  position: "absolute",
```

Comment:
This seems to work decently to me, but might have unintended consequences – the goal is to have a pop-up menu floating directly beneath where the button to open it is